shared/api/event.go

```diff
@@ -28,13 +29,60 @@ type Event struct {
 	Location string `yaml:"location,omitempty" json:"location,omitempty"`
 }
 
+// EventLogRecord represents single log record
+type EventLogRecord struct {
+	Time time.Time
+	Lvl  string
+	Msg  string
+	Ctx  []interface{}
+}
+
+// EventLogger is used for event logging purpose
+type EventLogger interface {
+	LogRecord(Event) (EventLogRecord, error)
+}
+
+// NewEventLogger creates specific instance of EventLogger interface
+func NewEventLogger(event Event) EventLogger {
```

Comment:
I think it'd be cleaner if one could do `event.ToLogging()` and get the LogRecord struct or an error if it's an unsupported type.

So maybe the cleaner approach here is to:
 - Add a `ToLogging() EventLogRecord, error` function on the `Event` struct.
 - Have that function check the type and return the log record or an error if unsupported.

Then in the CLI you can just call `.ToLogging()` if `--pretty` is passed.